index.html

```diff
@@ -231,16 +231,44 @@ <h2>
         Interfacing with a payment handler
       </h2>
       <p>
-        The <dfn>steps to constrain a payment handler</dfn> with
+        The <dfn>steps to filter payment handlers</dfn> with
```

Comment:
We probably want to call this "steps to check instrument support". We are not filtering "payment handlers", but rather, the instruments by capabilities.